src/components/roadmapList/Pagination.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import style from './RoadmapList.module.css';
+
+import Icon from 'components/ui/Icon';
+
+interface Props {
+  totalPages?: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+}
+
+const Pagination: React.FC<Props> = ({
+  totalPages = 10,
+  currentPage = 1,
+  onPageChange,
+}) => {
+    const getPageNumbers = () => {
+        const pageNumbers:number[] = [];
+        if (totalPages <= 3) {
+          for (let i = 1; i <= totalPages; i++) {
+            pageNumbers.push(i);
+          }
+        } else {
+          if (currentPage <= 2) {
+            for (let i = 1; i <= 3; i++) {
+              pageNumbers.push(i);
+            }
+          } else if (currentPage >= totalPages - 1) {
+            for (let i = totalPages - 2; i <= totalPages; i++) {
+              pageNumbers.push(i);
+            }
+          } else {
+            for (let i = currentPage - 1; i <= currentPage + 1; i++) {
+              pageNumbers.push(i);
+            }
+          }
+        }
+        return pageNumbers;
+      };
+
+  const handleIncrease =()=>{
+    if(currentPage === totalPages) return
+    onPageChange(currentPage + 1)
+
+  }
+
+  const handleDecrease =()=>{
+    if(currentPage === 1) return
+    onPageChange(currentPage - 1)
+  }
+
+  const handleNumberClick = (value:number)=>{
+    onPageChange(value)
+  }
+
+  return (
+    <div className={style.pagination}>
+      <button
+        onClick={handleDecrease}
+        disabled={currentPage === 1}
+        className={style.pagination_button}
+      >
+        <Icon id="icon-dropdown" fill="#2E3249" width={28} height={28} />
```

Comment:
The Icon component takes the width and height values, which is more readable than the size, so it is better to use it that way, because you are writing code for programmers who will read this code

Reply:
size property is a valid css style name, which is used constantly in many projects, i don't see a reason to make your code bigger if you have the same clear name. In general, width and height were added for cases where you have icons which have different width and height values

Reply:
I have an options write width and height of icon, and i choose it. It is not defect)